lib/modules/platform/bitbucket/index.ts

```diff
@@ -900,6 +917,26 @@ export async function createPr({
   }
 }
 
+async function effectiveDefaultReviewers(
```

Comment:
candidate to move to utils/ common, because file is already pretty big

Reply:
do we want http calls within utils.ts, too?

Reply:
What about if I consolidated the Reviewer related functions together? 🤔